package.json

```diff
@@ -46,19 +46,19 @@
     "core-js": "^3.25.2",
     "d3-array": "^2.12.0",
     "d3-axis": "^2.1.0",
-    "d3-brush": "^1.1.6",
-    "d3-color": "^1.4.0",
+    "d3-brush": "^3.0.0",
+    "d3-color": "^3.1.0",
     "d3-drag": "^1.2.4",
     "d3-force": "^2.1.1",
     "d3-hierarchy": "^1.1.9",
-    "d3-interpolate": "^2.0.1",
+    "d3-interpolate": "^3.0.0",
     "d3-path": "^1.0.9",
     "d3-quadtree": "^1.0.6",
-    "d3-scale": "^3.3.0",
+    "d3-scale": "^4.0.0",
     "d3-selection": "^1.4.2",
     "d3-shape": "^1.3.5",
-    "d3-transition": "^1.2.0",
-    "d3-zoom": "^1.8.3",
+    "d3-transition": "^3.0.0",
```

Comment:
I am not sure if this will bring any api change. Our tests for d3 widgets are not in-depth. Will cherry pick and test the changes first before merging.

Reply:
This will break, 100%. The only reason its compiling is the types weren't updated.

@rish691 please disable your snyk integration for this repo.

Reply:
I had disabled it, let me check.